python/lsst/ts/dsm/utils.py

```diff
@@ -22,60 +22,83 @@ def convert_time(in_time):
     float
         The TAI time corresponding to the input.
     """
-    ptime = Time(in_time, scale='utc')
+    ptime = Time(in_time, scale="utc")
     # replace allows this to work if clock isn't in UTC.
     return ptime.tai.datetime.replace(tzinfo=pytz.utc).timestamp()
 
 
-def create_telemetry_config(output_dir):
+def create_telemetry_config(output_dir, sim_loop_time):
     """Create the DSM UI Configuration file for simulation mode.
 
     Parameters
     ----------
     output_dir : `str`
         Directory to write the telemetry UI configuration file.
+    sim_loop_time : in or float
+        Description
     """
-    ui_version = '1.0.1'
-    ui_config_version = '1.4.4'
-    ui_config_file = '/dsm/ui_dsm_config/default.yaml'
-    camera_name = 'Vimba'
-    camera_fps = 40
-    data_buffer_size = 1024
-    data_acquisition_time = 25
-
-    content = {'timestamp': Time.now().isot,
-               'ui_versions': {'code': ui_version, 'config': ui_config_version,
-                               'config_file': ui_config_file},
-               'camera': {'name': camera_name,
-                          'fps': camera_fps},
-               'data': {'buffer_size': data_buffer_size,
-                        'acquisition_time': data_acquisition_time}}
-
-    filename = os.path.join(output_dir, 'dsm_ui_config.yaml')
-    with open(filename, 'w') as ofile:
+    ui_version = "1.0.1"
+    ui_config_version = "1.4.4"
+    ui_config_file = "/dsm/ui_dsm_config/default.yaml"
+    camera_name = "Sim_Camera"
+    if sim_loop_time > 1:
+        camera_fps = 40
+        data_buffer_size = 1024
+    else:
+        camera_fps = 120
+        data_buffer_size = 128
+    data_acquisition_time = sim_loop_time
+
+    content = {
```

Comment:
What you have works, but I gently suggest that dict's keyword constructor would make it a bit easier to read:
```
content = dict(
    timestamp = Time.now().isot,
    ui_versions = dict(
        code = ui_version,
        config = ui_config_version,
        config_file = ui_config_file,
    ),
    camera = dict(name=camera_name, fps=camera_fps),
    data = dict(buffer_size=data_buffer_size, acquisition_time=data_acquisition_time),
)
```

Reply:
Done.